airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -127,6 +128,15 @@ def execute(self, context):
                                                  cluster_context=self.cluster_context,
                                                  config_file=self.config_file)
 
+            # Add Airflow Version to the label
+            # And a label to identify that pod is launched by KubernetesPodOperator
+            self.labels.update(
+                {
+                    'airflow_version': airflow_version.replace('+', '-'),
```

Comment:
Does Kube recommend that we qualify our labels, like `org.apache.airflow.version`?

Reply:
Or is that for annotations? When should we use one over another?

Reply:
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ recommend some but given we already use labels and don't follow such conventions, I made this change :

https://github.com/apache/airflow/blob/fab957e763f40bf2a2398770312b4834fbd613e1/airflow/kubernetes/worker_configuration.py#L372-L378

Reply:
Re. labels vs annotations:

>You can use either labels or annotations to attach metadata to Kubernetes objects. Labels can be used to select objects and to find collections of objects that satisfy certain conditions. In contrast, annotations are not used to identify and select objects. The metadata in an annotation can be small or large, structured or unstructured, and can include characters not permitted by labels.
So labels make more sense for e.g to select all the pods that are created by Kube Pod Operator